cve_bin_tool/helper_script.py

```diff
@@ -214,7 +214,9 @@ def search_version_string(self, matched_list: list[str]) -> list[str]:
     def parse_filename(self, filename: str) -> tuple[str, str]:
         """
         returns package_name/product_name from package_filename of types .rpm, .deb, etc.
-        Example: package_filename = openssh-client_8.4p1-5ubuntu1_amd64.deb
+
+        Example: 
```

Comment:
```suggestion
        Example:
```

Removing trailing whitespace